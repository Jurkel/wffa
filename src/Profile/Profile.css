.player-name {
  position: relative;
  max-width: 60%;
  margin-right: auto;
  margin-left: auto;
  bottom: 37px;
  height: auto;
  border: 1px solid #FFD08A;
  border-radius: 5px;
  font-size: 1.1em;
  letter-spacing: .2em;
  text-transform: uppercase;
  color: #F2F2F2;
  background-color: rgba(255, 208, 138, .13);
  padding: 5px 0;
}

.player-title {
  font-family: 'Courier';
  font-size: 1.4em;
  color: #F2F2F2;
  text-align: center;
}

.carousel__dot-group {
  position: absolute;
  bottom: -30px;
  left: 50%;
  transform: translate(-50%, -50%);
}

.carousel__dot {
  margin: 3px;
  background-color: #F2F2F2;
  border: 1px solid #F2F2F2;
  border-radius: 15px;
}

.carousel__dot:active, .carousel__dot:hover {
  background-color: transparent;
}

.carousel {
  height: 450px;
  display: relative;
}

.player-info {
  height: 500px;
}