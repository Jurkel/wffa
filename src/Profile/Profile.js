import React from 'react'
import player from '../images/player.png'
import Bio from '../Bio/Bio'
import Stats from '../Stats/Stats'
import Starters from '../Starters/Starters'
import { CarouselProvider, Slider, Slide, DotGroup } from 'pure-react-carousel'
import 'pure-react-carousel/dist/react-carousel.es.css'
import './Profile.css'

class Profile extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      user: ''
    }
  }

  render() {
    return (
      <div className='roster-profile'>
        <div className='player-wrapper'>
          <img className='player-image' src={player} alt='dynamic player name' />
          <div className='player-name'></div>
        </div>
        <div className='player-info'>
          <CarouselProvider
            naturalSlideWidth={100}
            naturalSlideHeight={125}
            totalSlides={3}
          >
            <Slider>
              <Slide index={0}>
                <Bio />
              </Slide>
              <Slide index={1}>
                <Stats />
              </Slide>
              <Slide index={2}>
                <Starters />
              </Slide>
            </Slider>
            <DotGroup />
          </CarouselProvider>
        </div>
      </div>
    )
  }
}

export default Profile;