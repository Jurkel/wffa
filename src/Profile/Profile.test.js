import { shallow } from "enzyme";
import React from "react";
import Profile from "./Profile";

it("renders Profile component", () => {
  const mockProps = {
    location: {
      state: {
        managerBio: {
          owner_id: "594996794933477376",
          league_id: "590186196781543424",
          firstname: "Laila",
          lastname: "Fitriana",
          height: "5'3",
          weight: 123,
          school: "OSU",
          favoriteteam: "Ohio State University",
          favoriteplayer: "Zeke Elliot",
          experience: "1st year",
          display_name: "lailapfit",
          photolink: "../images/Laila.png",
          active: true,
          dob: "04-25-1990",
        },
        rosterStats: {},
      },
    },
    starters: {},
  };
  expect(shallow(<Profile props={mockProps} />)).toMatchSnapshot();
});