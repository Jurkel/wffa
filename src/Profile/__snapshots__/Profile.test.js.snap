// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Profile component 1`] = `ShallowWrapper {}`;
